compiler/testData/codegen/box/syntheticAccessors/accessorForProtectedPropertyReference.kt

```diff
@@ -0,0 +1,15 @@
+// FILE: 1.kt
+package a
+
+open class A {
+    protected val v = "OK"
+}
+
+// FILE: 2.kt
+import a.*
+
+class B : A() {
+    fun foo() = ::v.get()
```

Comment:
And just as I published this I realised that `B::v.get(this)` is still broken. Oh well.